man/cppcheck.1.xml

```diff
@@ -516,6 +615,62 @@ There are false positives with this option. Each result must be carefully invest
           </para>
         </listitem>
       </varlistentry>
+      <varlistentry>
+        <term>
+          <option>--premium=&lt;type&gt;</option>
+        </term>
+        <listitem>
+          <para>Enable premium analysis features. Available types:</para>
+          <variablelist>
+            <varlistentry>
+              <term>bughunting</term>
+              <listitem>
+                <para>Enhanced bug detection with higher false positive rate</para>
+              </listitem>
+            </varlistentry>
+            <varlistentry>
+              <term>autosar</term>
+              <listitem>
+                <para>Check AUTOSAR coding guidelines</para>
+              </listitem>
+            </varlistentry>
+            <varlistentry>
+              <term>cert-c</term>
+              <listitem>
+                <para>Check CERT C coding standard</para>
+              </listitem>
+            </varlistentry>
+            <varlistentry>
+              <term>cert-c++</term>
+              <listitem>
+                <para>Check CERT C++ coding standard</para>
+              </listitem>
+            </varlistentry>
+            <varlistentry>
+              <term>misra-c++-2008</term>
```

Comment:
lots of missing premium terms: 

    https://github.com/danmar/cppcheck/blob/main/cli/cmdlineparser.cpp#L1012